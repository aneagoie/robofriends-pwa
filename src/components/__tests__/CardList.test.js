import { shallow, mount, render } from "enzyme";
import React from "react";
import CardList from "../CardList";

test("Should render CardList component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      email: "john@email.com",
    },
  ];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
