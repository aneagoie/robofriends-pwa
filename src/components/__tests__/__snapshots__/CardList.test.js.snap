// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render CardList component 1`] = `
<div>
  <Card
    email="john@email.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
